mapper/orm-coordination-outbox-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/event/impl/OutboxPollingOutboxEventAdditionalJaxbMappingProducer.java

```diff
@@ -64,6 +64,7 @@
 			"        <property name=\"payload\" type=\"materialized_blob\" />\n" +
 			"        <property name=\"retries\" type=\"integer\" />\n" +
 			"        <property name=\"processAfter\" type=\"Instant\" index=\"processAfter\" />\n" +
+			"        <property name=\"status\" nullable=\"false\" />\n" +
```

Comment:
I just realized you probably want an index on this property, since you almost always filter on it when you run queries.